rfc/async-first-team.md

```diff
@@ -0,0 +1,92 @@
+4. Coordination Challenges: Coordinating tasks and resolving conflicts may require more effort and time in an async communication environment compared to synchronous methods.
+
+## Async first team process proposed
+
+Our team's characteristics naturally lend themselves to asynchronous communication, yet we recognize the value of synchronous interactions in certain contexts. Synchronous communication becomes particularly beneficial when expediting decision-making, clarifying objectives, and involving a small set of individuals. It's also important to maintain a sense of enjoyment and proportional connections within the team.
+
+We advocate for initiating asynchronous processes, which may culminate in synchronous sessions to reach consensus among team members, starting with a proposal. Following one or more rounds of asynchronous feedback and discussion, a subset of the team CAN be involved on a highly productive synchronous session to align on specific set of items.
+
+For fast final alignments and swift decision-making, it's essential to involve a small subset of the team in high-bandwidth one-on-one or small group synchronous discussions, prioritizing efficiency and flexibility.
+
+Considering the small intersection window of some members of the team, spontaneous synchronous discussions should be encouraged and recognized as they naturally occur, rather than relying solely on scheduled calendar slots that may hinder important conversations. While these spontaneous discussions MAY not always align immediately, it's advisable to plan them with some lead time before critical decisions need to be made.
+
+For sprint planning and ensuring everyone grasps the broader picture, a recurring weekly synchronous meeting may be suggested as a best practice. These meetings should be recorded, have a predefined agenda, and clearly outlined objectives from the outset.
+
+Whether recurring or impromptu, outcomes from meetings such as recordings, notes, or summaries should be shared with the broader team to ensure transparency and alignment.
+
+## Tips to thrive in such environment
+
+1. Continuously anticipate potential blockers or issues and communicate them proactively via asynchronous channels with the team before they impede progress.
+2. If you encounter a blockage, promptly notify the team through asynchronous communication and identify individuals who MAY offer assistance.
+3. For design work/team process changes, initiate the process with an asynchronous proposal by one or more team members. Subsequently, gather asynchronous feedback rounds to address simpler aspects.
+4. Prior to scheduling a meeting, ensure that all relevant materials have been shared in advance to allow the team to review and provide initial asynchronous feedback. This optimizes synchronous meeting time for aligning on complex issues rather than rehashing previously discussed topics.
```

Comment:
Couldn't agree more 👍. If there are pre-materials, then it's easier to contribute effectively